readme/Readme.scalatex

```diff
@@ -463,31 +559,147 @@
             @p
                 PythonParse is currently compatible enough to parse all the python sources in Zulip, Ansible, Changes, Django, and Flask. It isn't published yet on maven central, but feel free to look at it if you want an idea of how to write a complex, real parser.
 
+        @sect{CssParse}
+            @div(id := "cssdiv")
+            @script(raw("""demo.DemoMain().css(document.getElementById("cssdiv"))"""))
+
+            @p
+                @a("One more", href:="https://github.com/lihaoyi/fastparse/tree/master/cssparse/shared/src/main/scala/cssparse")
+                too big for describing parser that parses CSS files and prints them adding the necessary spaces and tabs.
+                In other words, making the file readable and good-looking.
+                In process it builds internal AST that stores information about tags and rules in the given CSS,
+                this AST isn't complete, because of complexity of initial CSS format,
+                but it provides all the essential information about basic elements of file (blocks, selectors, rules).
+                The correctness is tested by parsing and then printing several huge files including CSS from Bootstrap and Primer.
+
+        @sect{BmpParser}
+            @div(id := "bmpdiv")
+            @script(raw("""demo.DemoMain().bmp(document.getElementById("bmpdiv"))"""))
+            @i
+                Given form accepts BMP files and prints short info about it (width, height and etc.).
+                If you haven't any bmp around, you can download classic
+                @a("lena", href:="https://raw.githubusercontent.com/lihaoyi/fastparse/master/byteparse/jvm/src/test/resources/lena.bmp").
+
+            @p
+                @hl.scala{BmpParser} is a good first example of byte parser, it's not small, but structure is pretty simple.
+
+            @i
+                @hl.scala{AnyWordI} and @hl.scala{AnyDwordI} are just parsers that consumes any word (2 bytes) or dword (4 bytes).
+
+            @hl.ref(bytemain/"BmpParser.scala", "val fileHeader")
```

Comment:
Is `AnyWordI` and `AnyDWordI` part of the public API? If not, we should make them part of the public API and describe them under `Writing Byte Parsers`. An alternative is to define them directly in `BmpParser.scala`, or rip them out of `BmpParser.scala`  and replace them with the respective `.!` parsers